mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -431,7 +432,13 @@ async fn request_and_fill_orderbook(ctx: &MmArc, base: &str, rel: &str) -> Resul
             },
             _peer_id,
         )) => (pubkey_orders, protocol_infos, conf_infos),
-        None => return Ok(()),
+        None => {
+            if i_am_seed {
+                return Ok(());
```

Comment:
nit: Maybe we should log a warning here or something, what do you think?

Reply:
[0e809b4](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2318/commits/0e809b429e039902713085920777b9b7c40b6e05)